modules/gridmap/doc_classes/GridMap.xml

```diff
@@ -197,7 +182,7 @@
 	</methods>
 	<members>
 		<member name="bake_navigation" type="bool" setter="set_bake_navigation" getter="is_baking_navigation" default="false">
-			If [code]true[/code], this GridMap bakes a navigation region.
+			If [code]true[/code], this GridMap creates a navigation region for each cell that uses a mesh_library item with a navmesh. The create navigation region will use the [code]navigation_layers[/code] bitmask assigned to the mesh_library item.
```

Comment:
```suggestion
			If [code]true[/code], this GridMap creates a navigation region for each cell that uses a [member mesh_library] item with a navmesh. The created navigation region will use the [MeshLibrary]'s navigation layers bitmask defined using [method MeshLibrary.set_item_navigation_layers].
```
Tentative change to take into account what I described for the MeshLibrary doc change. I don't know if that's actually accurate.

Reply:
Preferred solution would be to just remove the code tag and use a plain "navigation layers" text.

Not sure the MeshLibrary set_item_navigation_layers() should be mentioned here as this is usually something users set with the inspector gui for the mesh library and not by script.